securing_apps/topics/oidc/java/servlet-filter-adapter.adoc

```diff
@@ -91,10 +91,6 @@ karaf@root()> feature:repo-add mvn:org.keycloak/keycloak-osgi-features/{project_
 karaf@root()> feature:install keycloak-servlet-filter-adapter
 ----
 
-For other OSGi runtimes, please refer to the runtime documentation on how to install the adapter bundle and its dependencies.
-
-NOTE: If your OSGi platform is Apache Karaf with Pax Web, you should consider using <<_fuse_adapter,JBoss Fuse 6>> or <<_fuse7_adapter,JBoss Fuse 7>> adapters instead.
```

Comment:
Should this be rewritten by saying that Pax Web adapter is maintained in RHSSO 7.x?

```suggestion
NOTE: If your OSGi platform is Apache Karaf with Pax Web, you should consider using https://access.redhat.com/products/red-hat-single-sign-on[Red Hat Single Sign-On] 7.x JBoss Fuse 7 adapters instead.
```

Reply:
Updated as suggested (couldn't do it through GH so applied manually)